hack/common.sh

```diff
@@ -122,10 +122,22 @@ run_WMCO() {
       fi
   fi
 
-  # Add the "openshift.io/cluster-monitoring:"true"" label to the operator namespace to enable monitoring
-  if ! oc label ns $WMCO_DEPLOY_NAMESPACE openshift.io/cluster-monitoring=true --overwrite; then
-      return 1
-  fi
+  # required labels for WMCO namespace
+  declare -a NS_LABELS=(
```

Comment:
Why is this being done here and not in the namespace we are creating when installing the operator:
https://github.com/openshift/windows-machine-config-operator/blob/master/config/manager/manager.yaml#L1

How is this going to work for actual users and not just CI?

Reply:
I dont understand why in general we create the namespace before attempting to install the operator. The only thing i can see is that we want to create the cloud-private-key secret ahead of time for some reason. I dont see that as necesary

Reply:
@sebsoto  you are correct. As per standup, this is out of the scope in this PR and should be tackled in a follow-up story.

Reply:
> I dont understand why in general we create the namespace before attempting to install the operator. The only thing i can see is that we want to create the cloud-private-key secret ahead of time for some reason. I dont see that as necesary

After attempting to install the operator without creating the namespace, got:
```shell
FATA[0000] Failed to run packagemanifests: create catalog: error creating catalog source: namespaces "openshift-windows-machine-config-operator" not found 
Error from server (NotFound): namespaces "openshift-windows-machine-config-operator" not found

```

Reply:
That's correct when deploying the operator using the official deployment method using the CatalogSource along with an operator index. 

However, for the e2e tests ([run-ci-e2e-test.sh](https://github.com/openshift/windows-machine-config-operator/blob/master/hack/run-ci-e2e-test.sh#L94)) the operator is deployed using OLM where the namespace is [created](https://github.com/openshift/windows-machine-config-operator/blob/master/hack/common.sh#L119) first  followed by the creation of the `cloud-private-key` secret. Therefore, the Pod Security Admission labels are required in this step.